src/DynamoCoreWpf/VisualizationManager/VisualizationManager.cs

```diff
@@ -534,6 +528,67 @@ protected virtual void HandleRenderPackagesReadyCore()
         }
 
         #endregion
+
+        /// <summary>
+        /// Use the renderPackageFactoryAssembly tag in the DynamoCore.dll.cfg file to find an
+        /// IRenderPackageFactory implementation.
+        /// </summary>
+        private IRenderPackageFactory GetRenderPackageFactory()
+        {
+            var renderPackageFactoryAsm = Dynamo.Utilities.AssemblyConfiguration.Instance.GetAppSetting("renderPackageFactoryAssembly");
+
+            if (string.IsNullOrEmpty(renderPackageFactoryAsm))
+            {
+                return CreateDefaultRenderPackageFactory();
+            }
+
+            var asmPath =
+                Path.Combine(
+                    Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
+                    renderPackageFactoryAsm);
+
+            if (!File.Exists(asmPath))
+            {
+                throw new Exception("The specified render package factory assembly does not exist.");
+            }
+
+            var asm = Assembly.LoadFrom(asmPath);
+
+            var factoryType = asm.GetTypes().FirstOrDefault(t => typeof(IRenderPackageFactory).IsAssignableFrom(t));
+            if (factoryType == null)
+            {
+                throw new Exception("An implementation of IRenderPackageFactory could not be found.");
+            }
+
+            // Construct the factory using the default constructor.
+            var factoryConstructor = factoryType.GetConstructor(System.Type.EmptyTypes);
+            var factory = (IRenderPackageFactory)factoryConstructor.Invoke(null);
+            factory.MaxTessellationDivisions = MAX_TESSELLATION_DIVISIONS_DEFAULT;
+
+            return factory;
```

Comment:
Must we dynamically load this?  Also, why are you overriding MaxTessellationDivisions divisions here? This would be hard for an IRenderPackageFactory implementor to reverse...